WordPress/Classes/ViewRelated/Blog/Blog Dashboard/ViewModel/BlogDashboardViewModel.swift

```diff
@@ -147,16 +147,7 @@ private extension BlogDashboardViewModel {
 
     func applySnapshot(for cards: [DashboardCardModel]) {
         let snapshot = createSnapshot(from: cards)
-        let scrollView = viewController?.mySiteScrollView
-        let position = scrollView?.contentOffset
-
-        dataSource?.apply(snapshot, animatingDifferences: false) { [weak self] in
-            guard let scrollView = scrollView, let position = position else {
-                return
-            }
-
-            self?.scroll(scrollView, to: position)
```

Comment:
The `scroll(_:to:)` code was added in the [following PR](https://github.com/wordpress-mobile/WordPress-iOS/pull/18245) as a workaround. Unfortunately, it breaks the new personalization feature (see the [comment](https://github.com/wordpress-mobile/WordPress-iOS/pull/20369#pullrequestreview-1357343984)).

I tried removing the workaround to see if it's still needed and was not able to reproduce the issue it's supposed to be fixing.